src/include/stir/IO/InputStreamWithRecords.h

```diff
@@ -125,10 +125,13 @@ class InputStreamWithRecords
   inline
     void set_saved_get_positions(const std::vector<std::streampos>& );
 
-private:
+  inline
+  shared_ptr<std::istream> get_stream_sptr(){return this->stream_ptr;}
```

Comment:
Why do we need this? If we do, it'd be safer to have `std::istream& get_stream()`

Reply:
because the get_save_position and those similar functions where not working with my change of folder for the Mediso scanner, and I had to do it via the istream functions.